internal/service/secretsmanager/secret_version_test.go

```diff
@@ -271,6 +274,39 @@ func TestAccSecretsManagerSecretVersion_multipleVersions(t *testing.T) {
 	})
 }
 
+func TestAccSecretsManagerSecretVersion_stringWriteOnly(t *testing.T) {
+	ctx := acctest.Context(t)
+	var version secretsmanager.GetSecretValueOutput
+	rName := sdkacctest.RandomWithPrefix(acctest.ResourcePrefix)
+	resourceName := "aws_secretsmanager_secret_version.test"
+	secretResourceName := "aws_secretsmanager_secret.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(ctx, t); testAccPreCheck(ctx, t) },
+		ErrorCheck:               acctest.ErrorCheck(t, names.SecretsManagerServiceID),
```

Comment:
Oops - missed this version check on the previous pass.

```suggestion
		ErrorCheck:               acctest.ErrorCheck(t, names.SecretsManagerServiceID),
		TerraformVersionChecks: []tfversion.TerraformVersionCheck{
			tfversion.SkipBelow(version.Must(version.NewVersion("1.11.0"))),
		},
```